### DISCUSSION TITLE
Is this package compatible with spatie/laravel-view-models

### DISCUSSION THREAD

**Discussion by Bloemendaal** [spatie/laravel-typescript-transformer#21] [General]
I recently started using this package, but I was still writing the React props of the page myself. Then I figured, why not try to put `#[TypeScript]` above my ViewModel, but this doesn't seem to work. Am I missing something, or is it just not possible?

**Comment by rubenvanassche** [comment]
You will have to write a transformer for this since we cannot automatically deduce what the structure is.
